Cython/Utility/ModuleSetupCode.c

```diff
@@ -256,7 +256,7 @@
   #undef CYTHON_USE_FREELISTS
   #define CYTHON_USE_FREELISTS 0
 
-#elif defined(Py_GIL_DISABLED) || defined(Py_NOGIL)
+#elif defined(Py_NOGIL)
```

Comment:
Now that the nogil fork is merged, it probably makes sense to treat it as a special case of CPython. The risky sections that we used to hide under the `…IN_CPYTHON` macro should long have received their own special macro (like `USE_BORROWED_REFS` etc.).